doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -1,10 +1,27 @@
-[id='rn-known-issues-con']
-= Known issues
+[id='rn-known-issues-ref']
+= Known issues in {PRODUCT} 7.7.0
 
 This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
 == {CENTRAL}
 
+
+.In a high-availability deployment of {CENTRAL}, deploying a service to a {KIE_SERVER} using {CENTRAL} intermittently fails [https://issues.redhat.com/browse/RHPAM-2830[RHPAM-2830]]
+
+Issue: In a high-availability deployment of {CENTRAL}, when you use the {CENTRAL} user interface to deploy a service to a {KIE_SERVER}, sometimes the service is not deployed. At other times the service might be deployed successfully.
+
+Workaround: In an on-premise deployment using {EAP}, there is no workaround. Do not use an on-premises high-availability deployment of {CENTRAL} to deploy services to {KIE_SERVERS}.
```

Comment:
Suggested rewrite:
There is no workaround in an on-premise deployment using the {EAP}. Do not use...